### ISSUE TITLE
Add a about us link on hamburger menu for android 

### ISSUE THREAD

**Issue by bhujoshi** [croma-app/croma-react#101]
Content for about us screen.

- Add an intro - Croma is a simple color palette manager and color picker made for designers, aiming to make it quick and fun to create and share color palettes on the go.
- And a Github link - https://github.com/croma-app/croma-react
- And a Feedback and suggestion link - https://github.com/croma-app/croma-react/issues/new

Add a new menu item just below of `import/export palettes`. 
![menu](https://user-images.githubusercontent.com/9627563/95660894-86ad7e80-0b48-11eb-9b34-5b4903d4fadb.jpeg)

**Comment by EduardoFLima** [comment]
I would like to work on this. Knowing objective-c is a must ? because I have never work with.

**Comment by Hirak-Pandya** [comment]
Can I work on this issue?

**Comment by Yashs911** [comment]
Hi @bhujoshi I would like to work on this

**Comment by bhujoshi** [comment]
Thanks, guys assigning this issue to @Yashs911. Let me know If I can help you with other issues @Hirak-Pandya and @EduardoFLima

**Comment by bhujoshi** [comment]
@Yashs911 Please acknowledge you are working on it.

**Comment by Yashs911** [comment]
@bhujoshi Yes I am working on this issue.

**Comment by Yashs911** [comment]
@bhujoshi  can you please elaborate a little on what all changes do I need to make as there is already a github link
![Capture](https://user-images.githubusercontent.com/64412143/95659007-05e88580-0b3c-11eb-9edf-c53aed152169.JPG)

**Comment by bhujoshi** [comment]
@Yashs911 We need to add the About us screen only for the android app.

**Comment by bhujoshi** [comment]
@Yashs911 Add a new menu item just below of `import/export palettes`. 
![menu](https://user-images.githubusercontent.com/9627563/95660627-b78cb400-0b46-11eb-8978-7b6f98672592.jpeg)

**Comment by Yashs911** [comment]
@bhujoshi Sorry I don't think I would be able to solve this issue.

**Comment by Yashs911** [comment]
@bhujoshi I am currently learning web dev so....

**Comment by bhujoshi** [comment]
@Yashs911 No problem thanks for your time. @EduardoFLima You can take this up. objective-c knowledge not needed for this one. You will be able to do it using react-native.

**Comment by EduardoFLima** [comment]
Hey @bhujoshi ,
Mostly I have experience with react and not react native and the times I developed react native I just use expo to run the app. I'm trying to run croma-app and I noticed that is setup to run in a emulator or real device so I'm trying to run it in the emulator. Im having some problems with this set up, would you check if you faced this issue before?

I've setup firebase and followed [react native instructions](https://reactnative.dev/docs/environment-setup) for setting up the emulator.

I run the emulator - worked just fine;

I run `npm start`, the packager seems to run fine on port 8081 - I can even open in the browser;

Then, in other terminal, I run `npm run android`, the build is successful and it connects to the emulator and the app lunches there. However, an error is thrown: `Unable to load script. Make sure you're either running a Metro server (run 'react-native start') or that your bundle 'index.android.bundle' is packaged correctly for release.`

Then, I go back to the previous terminal and try to reload the app hitting its shortcut "r" but then I face the error `Sending "reload" to all React Native apps failed. Make sure your app is running in the simulator or on a phone connected via USB.`

Any ideas what it could be ? I already tried to reinstalled all SDKs again but nothing changed. Also, I tried to run the app with "bundle-and-run-android.sh" but it didn't work - it is saying `error ENOENT: no such file or directory, open 'android/app/src/main/assets/index.android.bundle'. Run CLI with --verbose flag for more details.`

It seems to me that the packager is not being able to connect to the emulator somehow but im not sure how to make sure of it..

**Comment by EduardoFLima** [comment]
I just tried to run with `expo start --android` instead of `npm start` and the app is opening now in the emulator however is throwing errors because `Platform.OS === "web"` is returning true when it shouldn't ..

**Comment by bhujoshi** [comment]
@EduardoFLima try to run ./bundle-and-run-android.sh I think then it will work.

**Comment by kamalkishor1991** [comment]
Hi @EduardoFLima , Did the above suggestion worked?. This is the first time anyone contributing to the repository so we are also trying to figure out gaps in the setup. Please let me know if you are still facing the issue or need any help? Thanks again for picking up the issue.

**Comment by EduardoFLima** [comment]
Hey @kamalkishor1991 ,
In the end I just had to create the assets directory (android/app/src/main/assets) and then it worked fine!
Apart from that I also had to setup firebase if I remember right. Later we can create another PR with some improvements in the readme.md adding further instructions of how to build the project.

I fixed that problem this afternoon and this weekend I believe I can go on with the task.

**Comment by EduardoFLima** [comment]
I still face the issue of Platform.OS always returning "web" instead of "android" though, causing exceptions...

**Comment by kamalkishor1991** [comment]
I always bundle and then run it in android. Since the project was an expo project earlier and then moved to react-native the normal development mode does not work. I will look into it. 
Is ``` ./bundle-and-run-android.sh ``` working? Please email me on kkjoshi.it@gmail.com and I can invite you to slack and we can find some time to debug it together.

**Comment by EduardoFLima** [comment]
I just realized that I forgot to mention that I also had to add the following config to android manifest in order to get connection between metro packager and the app in the emulator: 
```
<application
android:usesCleartextTraffic="true">
...
</application>
```

Yes, when I run ```bundle-and-run-android.sh``` works. But it takes too long, isn't annoying when developing ?

I will send the email. Thanks for the support.

**Comment by EduardoFLima** [comment]
Hey... Just FYI,

I misinterpreted the error. The problem was not in Platform.OS, that's working fine... The problem were in some StyleSheet.create() methods, which doesn't allow the property `position: "fixed"` when running the app in the android emulator. I refactored them so I could run the project and commit those changes.

I created this pull request #105 with the new feature but im not sure if I have credentials to link this issue to the PR.
